src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -111,9 +115,12 @@ export class TechRecordSummaryComponent implements OnInit {
         return this.getHgvTemplates();
       case VehicleTypes.TRL:
         return this.getTrlTemplates();
-      // TODO: Create light vehicle specific functions for this
```

Comment:
Is there anyway we can refactor this into a map

Reply:
You know I love maps

Reply:
Like why do we have a getter referencing getters, could we not have one getter that stores all of the template values in a map

Reply:
fine I guess!!!